### DISCUSSION TITLE
Node Groups

### DISCUSSION THREAD

**Discussion by sebastianhoerz** [Comfy-Org/ComfyUI_frontend#333] [Ideas]
Would love to have a section in the sidebar with Node-Groups i can predefine. Like saving a Group i did before with the settings still intact. It would save so many clicks!

**Comment by 0x473354** [comment]
Being able to distribute group nodes with custom nodes would also be quite useful, though it would probably require changes both to frontend and Comfy. In such a case I imagine being able to save your own group nodes would mostly be a subset of that functionality.
